packages/@jsii/utils/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please configure this for out-of-source building. You can see how `@jsii/spec` is doing it and replicate... This will involve renaming `/lib/` to `/src/`, too.